src/main/kotlin/dsm/wemeet/global/socket/WebSocketHandler.kt

```
@@ -0,0 +1,83 @@
+
+        val receiverSession = sessionMap[toUserId]
+        receiverSession?.takeIf { it.isOpen }?.sendMessage(TextMessage("Message received: $content"))
+    }
+
+    private fun userIdFromSession(session: WebSocketSession): String? {
+        val authorization = session.handshakeHeaders.getFirst("Authorization") ?: return null
+        val token = authorization.removePrefix("Bearer ").trim()
+        return try {
+            val claims = jwtProvider.getJws(token).body
+            claims.subject
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)